App/templates/register/signin.html.twig

```diff
@@ -1,7 +1,11 @@
 {% extends "base.html.twig" %}
+
+{% trans_default_domain 'registration' %}
```

Comment:
Si tu as un domain commun entre le log in et le sign up, peut être le renommer `authentication`